domains/kuro.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Kurorin-fs",
+        "twitter": "Kurorin_fs"
+    },
+    "record": {
+        "CNAME": "https://kurorin-fs.github.io/kurorin-fs"
```

CNAME can't contain protocol or `/`
```suggestion
        "CNAME": "kurorin-fs.github.io"
```